sdk/metric/metricdata/metricdatatest/assertion.go

```diff
@@ -53,6 +52,14 @@ type Datatypes interface {
 	// Aggregation and Value type from metricdata are not included here.
 }
 
+// TB is an interface common to `testing.T` and `testing.B` but without the
+// private method of `testing.TB`, so other testing packages can rely on it as
+// well.
+type TB interface {
```

Comment:
I suggest renaming to `T` or `TestingT`. As far as I remember this is a common pattern.

PS. You usually do not assert in benchmarks. But you assert in fuzz tests.

Reply:
Sure, though the pattern isn't used in the standard lib hence me naming it TB originally.